kafka/tools/create_volumes.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ux
+
+model=`juju show-model --format=json| jq -r '.| keys[]'`
+machines=`juju status $model --format=json | jq -r '.machines | to_entries[].value."instance-id"'`
+num=`echo "$machines" | wc -l`
+
+for i in $(seq 1 $((num*2))); do
+	if [ "`openstack volume list --name kafka-vol-$i -c Status -f value`" != "available" ]; then
```

Comment:
If one of the volumes is already in use you would create another volume with the same name, which would lead to a failure in lines 15 and 16 because you are resolving the volume there by name and not ID.